### PR TITLE
Update changelog for v0.13.0

### PR DESCRIPTION
**Pull request by radhikaj**
Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

Will rebase if  https://github.com/openenclave/openenclave/pull/3768 is merged first.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased][Unreleased_log]
 --------------
 
+[v0.13.0][v0.13.0_log]
+--------------
+
```
Comment (yentsanglee) [inline CHANGELOG.md:15]: ### Added
- Enabled oe_verify_evidence() with a NULL format id to verify the legacy report generated by oe_get_report().
- Added the following SGX attestation claims from oe_verify_evidence():
     - OE_CLAIM_SGX_PF_GP_EXINFO_ENABLED
     - OE_CLAIM_SGX_ISV_EXTENDED_PRODUCT_ID
     - OE_CLAIM_SGX_IS_MODE64BIT
     - OE_CLAIM_SGX_HAS_PROVISION_KEY
     - OE_CLAIM_SGX_HAS_EINITTOKEN_KEY
     - OE_CLAIM_SGX_USES_KSS
     - OE_CLAIM_SGX_CONFIG_ID
     - OE_CLAIM_SGX_CONFIG_SVN
     - OE_CLAIM_SGX_ISV_FAMILY_ID
- Added the following fields for SGX KSS (Key Separation and Sharing) support:
     - FamilyID
     - ExtendedProductID

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
      - Enclave apps that are built with Make and rely on Open Enclave's pkgconfig must now explicitly include OE crypto wrapper library in linker dependency flags.
      - See the [Makefile in the helloworld sample](samples/helloworld/enclave/Makefile#L34) for an example. Here `OE_CRYPTO_LIB` is set to `mbedtls` in [parent MakeList file](samples/helloworld/Makefile#L9).
 
+
 ### Changed
 - Syscalls are internally dispatched directly to their implementation functions instead of via a switch-case.
```
Comment (yentsanglee) [inline CHANGELOG.md:26]: - Changed the attestation evidence extension OIDs for certificates generated by the following APIs. Verifiers must call oe_verify_attestation_certificate APIs from v.0.11.0 or above.
     - oe_generate_attestation_certificate(): "1.3.6.1.4.1.311.105.1" 
     - oe_get_attestation_certificate_with_evidnece(): "1.3.6.1.4.1.311.105.2"

**Comment by yentsanglee** [comment]
Please add my comments about attestation changes.

(These comments were from #3768 which I will abandon)

**Comment by radhikaj** [comment]
> Please add my comments about attestation changes.
> 
> (These comments were from #3768 which I will abandon)

No, please dont abandon your PR. we can get that merged first. I am doing a bors r+ on yours.

**Review by mingweishih — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mingweishih) [inline CHANGELOG.md:15]:
```
## Added
- OpenSSL version 1.1.1 libraries are now available for an enclave to use. See the [attested_tls sample](samples/attested_tls#build-and-run) for an example of building enclaves with OpenSSL.
```

File: CHANGELOG.md (hunk 2 above)
Comment (yentsanglee) [inline CHANGELOG.md:26]: Please add this bullet item to **Changed** section:

- Changed the attestation evidence extension OIDs for certificates generated by the following APIs. Verifiers must call oe_verify_attestation_certificate APIs from v.0.11.0 or above.
oe_generate_attestation_certificate(): "1.3.6.1.4.1.311.105.1"
oe_get_attestation_certificate_with_evidnece(): "1.3.6.1.4.1.311.105.2"

**Review by manojrupireddy — APPROVED** [review]

**Review by nonpolarity — APPROVED** [review]

**Review by yentsanglee — APPROVED** [review]

**Pull request merged**